Resources/Private/Forms/Contact.form.yaml

```diff
@@ -8,13 +8,10 @@ finishers:
         identifier: EmailToReceiver
         options:
             subject: '{subject}'
-            recipientAddress: ''
-            recipientName: ''
```

Comment:
These have been dropped without replacement?

Reply:
Just did the same mentioned in Slack. What should this look like?

```
recipients:
  '': ''
```

Reply:
As mentioned in Slack these options should be skipped in the first place and thus also do not need a migration. Users of the Bootstrap Package will never edit this file to add static values. Instead they will either copy it and add static values or set recipients via finisher overrides.

Reply:
Yes, agree.